Entities/Actor.cpp

```diff
@@ -1637,6 +1636,8 @@ void Actor::Draw(BITMAP *pTargetBitmap,
 
 void Actor::DrawHUD(BITMAP *pTargetBitmap, const Vector &targetPos, int whichScreen, bool playerControlled)
 {
+    m_HUDStack = -m_CharHeight / 2;
```

Comment:
You moved this to avoid it not happening cause of the early returns for when hud isn't visible/team is -1 right?
Please move the comment with it too for whoever eventually cleans up Actor